### ISSUE TITLE
Buttons break out of screen estate given space constraints

### ISSUE THREAD

**Issue by dneustadt** [shopware/platform#17223]
### Shopware Version

6.7.10.2

### Affected area / extension

Platform(Default)

### Actual behaviour

* Buttons exceed viewport width, cutting of button labels without text breaking

### Expected behaviour

* Given certain conditions button label text should be allowed to break to make buttons fully visible

### How to reproduce

Considering the constraints given by [WCAG S.C. 1.4.4](https://www.w3.org/WAI/WCAG22/Understanding/resize-text.html)

* Viewport width set to 320px
* Text zoom set to 200%
* Buttons break viewport width without text breaking

<img width="409" height="748" alt="Image" src="https://github.com/user-attachments/assets/3c4e31bb-06c6-47b4-abd5-eb906703cd83" />
